components/anchor/__tests__/Anchor.test.tsx

```diff
@@ -0,0 +1,341 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Anchor from '..';
+import { sleep } from '../../../tests/utils';
+
+const { Link } = Anchor;
+
+function createGetContainer(id: string) {
+  return () => {
+    const container = document.getElementById(id);
+    if (container == null) {
+      throw new Error();
+    }
+    return container;
+  };
+}
+
+function createDiv() {
+  const root = document.createElement('div');
+  document.body.appendChild(root);
+  return root;
+}
+
+let idCounter = 0;
+const getHashUrl = () => `API-${idCounter++}`;
```

Comment:
```suggestion
const getHashUrl = () => `Anchor-API-${idCounter++}`;
```

Will this be better?

Reply:
Almost the same